src/Schema/Events/index.ts

```diff
@@ -22,6 +22,9 @@ import {
   ClickedLoadMore,
   ClickedMainArtworkGrid,
   ClickedNavigationTab,
+  ClickedOnFrameMeasurements,
+  ClickedOnArtworkShippingUnitsDropdown,
```

Comment:
Does "clicked on" mean "focused", since they could tab to a dropdown? Do we care if they select a field or just that they focused on the field?